framework/docs/source/how-to-migrate-from-openfl.rst

```diff
@@ -0,0 +1,537 @@
+    fraction-evaluate = 0.5
+    local-epochs = 1
+    learning-rate = 0.1
+    batch-size = 32
+
+    ...
+
+Server-side Code
+~~~~~~~~~~~~~~~~~~~~~
+
+In OpenFL, all of the aggregator-side code is configured via the `plan.yaml` file through the
+specification of different arguments. In Flower, the exact tasks performed by the server are more
+configurable through code. For example, aggregation algorithms are added through a ``Strategy``,
+and the logic to save models is added explictly. Here is a ``ServerApp`` 
+(akin to an OpenFL Aggregator) compatible with the prior code snippets:
+
+.. code-block:: python
+
+    import torch
+    from flwr.app import ArrayRecord, ConfigRecord, Context, MetricRecord
+    from flwr.serverapp import Grid, ServerApp
+    from flwr.serverapp.strategy import FedAvg
+    
+    from pytorchexample.task import Net, load_centralized_dataset, test
+    
+    # Create ServerApp
+    app = ServerApp()
+    
+    
+    @app.main()
+    def main(grid: Grid, context: Context) -> None:
+        """Main entry point for the ServerApp."""
+    
+        # Read run config
+        fraction_evaluate: float = context.run_config["fraction-evaluate"]
+        num_rounds: int = context.run_config["num-server-rounds"]
+        lr: float = context.run_config["learning-rate"]
+    
+        # Load global model
+        global_model = Net()
+        arrays = ArrayRecord(global_model.state_dict())
+    
+        # Initialize FedAvg strategy
+        strategy = FedAvg(fraction_evaluate=fraction_evaluate)
+    
+        # Start strategy, run FedAvg for `num_rounds`
+        result = strategy.start(
+            grid=grid,
+            initial_arrays=arrays,
+            train_config=ConfigRecord({"lr": lr}),
+            num_rounds=num_rounds,
+            evaluate_fn=global_evaluate,
+        )
+    
+        # Save final model to disk
+        print("\nSaving final model to disk...")
+        state_dict = result.arrays.to_torch_state_dict()
+        torch.save(state_dict, "final_model.pt")
+
+You'll notice that this each ``ServerApp`` should have specific logic for working
+with a given deep learning framework (in this case PyTorch) due to the saving of a final model.
+This change requires only a few lines of modifications, and Flower has support for an extensive
+set of deep learning frameworks in it's `examples <https://github.com/adap/flower/tree/main/examples>`_ 
+(Tensorflow, FastAI, Huggingface, etc.) should you need reference code.
+
+Further help
+------------
+
+While we expect this guide will help most users get migrated quickly to the Flower ecosystem,
+certain complex OpenFL workloads may require more clarification or help. If you have further questions, 
+`join the Flower Slack <https://flower.ai/join-slack/>`_
+(and use the channel ``#questions``) or join our 
+`OpenFL Continuity Program <https://docs.google.com/forms/d/e/1FAIpQLScprGGX_jFRoEUv4HbJkkhkg6O7e5eCiq7uP95_0xK5Qnt1gA/viewform>`_
+to get in touch with our team!
+
```

Comment:
Maybe worth pointing readers to the [quickstart-pytorch-tutorial](https://flower.ai/docs/framework/tutorial-quickstart-pytorch.html) for a more detailed walkthrough the code generated by the PyTorch template? And potentially also point to [the longer tutorial](https://flower.ai/docs/framework/tutorial-series-get-started-with-flower-pytorch.html) for a much more step-by-step and in depth guide?